A71-xds-fallback.md

```diff
@@ -81,18 +81,26 @@ target.
 ### Update CSDS to aggregate configs from multiple XdsClient instances.
 
 The CSDS RPC service will be changed to get data from all XdsClient instances.
-A new string field named `channel_target` will be added to the CSDS
-`ClientConfig` message to indicate the channel target the data is associated
+A new string field named `client_scope` will be added to the CSDS
+[ClientConfig] message to indicate the channel target the data is associated
 with.
 
-Adding this field will not require updating the CSDS client as its value will
-be the same for most resources in most cases.
+```
+// For xDS clients, the scope in which the data is used. 
+// For example, gRPC indicates the data plane target or that the data is
+// associated with gRPC server(s).
+string client_scope = 4;
+```
+
+Adding this field will not require updating most CSDS clients as its value will
```

Comment:
s/its value will be the same for all resources/any given resource will have only a single client_scope/

Reply:
Thanks! Updated.